agent/app/agent_capability_unix.go

```diff
@@ -236,3 +237,19 @@ func (agent *ecsAgent) getTaskENIPluginVersionAttribute() (*ecs.Attribute, error
 func defaultIsPlatformExecSupported() (bool, error) {
 	return true, nil
 }
+
+// var to allow mocking for checkNetworkTooling
+var isNetworkToolingAvailable = checkNetworkTooling
```

Comment:
You'll probably need to implement a no-op/dummy function for non-linux as well.

Reply:
https://github.com/aws/amazon-ecs-agent/blob/master/agent/app/agent_capability_windows.go

https://github.com/aws/amazon-ecs-agent/blob/master/agent/app/agent_capability_unspecified.go

Reply:
Implemented in https://github.com/aws/amazon-ecs-agent/commit/1a3ea5b7f2b55d50e8b41c84c10a75842c262893